compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -2278,6 +2313,14 @@ protected BLangNode transformSyntaxNode(Node node) {
     }
 
     // ------------------------------------------private methods--------------------------------------------------------
+    BLangValueType addValueType(DiagnosticPos pos, String typeName) {
+        BLangValueType typeNode = (BLangValueType) TreeBuilder.createValueTypeNode();
+        typeNode.pos = pos;
+        typeNode.typeKind = (TreeUtils.stringToTypeKind(typeName.replaceAll("\\s+", "")));
```

Comment:
do we need `replaceAll()`?

Reply:
Additional parenthesis

Reply:
I think you don't even need the type name here. Why not pass the type kind directly?